### DISCUSSION TITLE
How to generate a list of consecutive numbers in Helix?

### DISCUSSION THREAD

**Discussion by chtenb** [helix-editor/helix#10746] [Q&A] [answered]
This use case comes up every now and then, where I need to generate a list of increasing numbers, e.g. one per line. What is the easiest way to do this in helix?

**Comment by masriomarm** [comment]
Multiple cursor select your number column and press in sequence.
`" C-# C-a`

**Reply by chtenb** [reply]
It doesn't work for me. I also don't think <C-#> is part of the default mapping?

**Reply by sibouras** [reply]
spawn multiple cursors then use the selection indices register(`#`)

**Reply by chtenb** [reply, marked as answer]
Oh, so you mean `"#P`? That's what works for me anyway :) Thanks for pointing to the # register.

**Reply by masriomarm** [reply]
Oh yes,my fault sorry, correct is `"# C-a`
That'll increment current number with the selection register.
If you don't yet have any numbers, you can `"#p` which will paste current selection index.
